internal/prompts/app_select.go

```diff
@@ -1162,7 +1162,9 @@ func flatAppSelectPrompt(
 			return SelectedApp{}, slackerror.New(slackerror.ErrInstallationRequired)
 		}
 	case ShowAllApps, ShowInstalledAndNewApps:
-		if manifestSource.Equals(config.ManifestSourceLocal) {
+		isManifestSourceLocal := manifestSource.Equals(config.ManifestSourceLocal)
+		isBoltInstallEnabled := clients.Config.WithExperimentOn(experiment.BoltInstall)
+		if isManifestSourceLocal || isBoltInstallEnabled {
```

Comment:
note: Updated the app select to display "create a new app" for both local and remote manifest sources. This means the `slack install` now displays "create a new app" for remote projects, even though it doesn't work. However, I think this is acceptable because is currently returns an error and the update continues to return an error. We'll follow-up this PR with support for `slack install`.